zone/spell_effects.cpp

```diff
@@ -8440,6 +8419,31 @@ bool Mob::HarmonySpellLevelCheck(int32 spell_id, Mob *target)
 	return true;
 }
 
+bool Mob::PassCharmTargetRestriction(Mob *target) {
```

Comment:
Can we add logs here?

Reply:
What would you like in your logs?

Reply:
at least something to indicate we failed because of this check, could help with potential bug reports (people can miss messages ..)

Reply:
Done